apps/web/src/pages/_document.tsx

```diff
@@ -0,0 +1,215 @@
+          />
+          <link
+            rel="apple-touch-startup-image"
+            href="/icons/apple-splash-1125-2436.png"
+            media="(device-width: 375px) and (device-height: 812px) and (-webkit-device-pixel-ratio: 3) and (orientation: portrait)"
+          />
+          <link
+            rel="apple-touch-startup-image"
+            href="/icons/apple-splash-2436-1125.png"
+            media="(device-width: 375px) and (device-height: 812px) and (-webkit-device-pixel-ratio: 3) and (orientation: landscape)"
+          />
+          <link
+            rel="apple-touch-startup-image"
+            href="/icons/apple-splash-1242-2688.png"
+            media="(device-width: 414px) and (device-height: 896px) and (-webkit-device-pixel-ratio: 3) and (orientation: portrait)"
+          />
+          <link
+            rel="apple-touch-startup-image"
+            href="/icons/apple-splash-2688-1242.png"
+            media="(device-width: 414px) and (device-height: 896px) and (-webkit-device-pixel-ratio: 3) and (orientation: landscape)"
+          />
+          <link
+            rel="apple-touch-startup-image"
+            href="/icons/apple-splash-828-1792.png"
+            media="(device-width: 414px) and (device-height: 896px) and (-webkit-device-pixel-ratio: 2) and (orientation: portrait)"
+          />
+          <link
+            rel="apple-touch-startup-image"
+            href="/icons/apple-splash-1792-828.png"
+            media="(device-width: 414px) and (device-height: 896px) and (-webkit-device-pixel-ratio: 2) and (orientation: landscape)"
+          />
+          <link
+            rel="apple-touch-startup-image"
+            href="/icons/apple-splash-1242-2208.png"
+            media="(device-width: 414px) and (device-height: 736px) and (-webkit-device-pixel-ratio: 3) and (orientation: portrait)"
+          />
+          <link
+            rel="apple-touch-startup-image"
+            href="/icons/apple-splash-2208-1242.png"
+            media="(device-width: 414px) and (device-height: 736px) and (-webkit-device-pixel-ratio: 3) and (orientation: landscape)"
+          />
+          <link
+            rel="apple-touch-startup-image"
+            href="apple-splash-750-1334.png"
+            media="(device-width: 375px) and (device-height: 667px) and (-webkit-device-pixel-ratio: 2) and (orientation: portrait)"
+          />
+          <link
+            rel="apple-touch-startup-image"
+            href="apple-splash-1334-750.png"
+            media="(device-width: 375px) and (device-height: 667px) and (-webkit-device-pixel-ratio: 2) and (orientation: landscape)"
+          />
+          <link
+            rel="apple-touch-startup-image"
+            href="apple-splash-640-1136.png"
+            media="(device-width: 320px) and (device-height: 568px) and (-webkit-device-pixel-ratio: 2) and (orientation: portrait)"
+          />
+          <link
+            rel="apple-touch-startup-image"
```

Comment:
Is there a better way instead of a bunch of link ?

Reply:
I don't know, I run the script:

```
npx pwa-asset-generator --xhtml --opaque false --type png logo.png
```

and its generated all tags for me, and I copied/pasted them on the file.